.github/workflows/build_all.yml

```diff
@@ -373,104 +373,196 @@ jobs:
           SLACK_ICON_EMOJI: ":heavy_check_mark:"
           SLACK_COLOR: "#00FF00"
           SLACK_FOOTER: ' '
-  agw-container-build:
+
+  agw-container-build-c-ghz:
+    runs-on: ubuntu-latest
     if: github.repository_owner == 'magma'
-    name: agw container build
+    # The dependency on the regular build is intentional.
+    # There is a hidden dependency in the dockerfile.
+    # As GHZ is not a production build, a more sophisitacted dependency chain
```

Comment:
Typo: `sophisticated`

Reply:
Do we really need lines 382/383 here? I think it would be enough to mention this in the PR description or so.

Reply:
Same for the other jobs